src/App.tsx

```diff
@@ -18,6 +18,12 @@ function App() {
     setList([...list]);
   };
 
+  const deleteTask = (index: number) => {
+    const items = JSON.parse(localStorage.getItem('todoList') || '[]');
```

Comment:
you shouldn't change this data... You should just modify you `list` and when you modify it, your 
```
  useEffect(() => {
    localStorage.setItem('todoList', JSON.stringify(list));
  }, [list]);
` ``
will save it